twilio/twiml/__init__.py

```diff
@@ -120,3 +120,21 @@ def xml(self):
                 el.append(last_child)
 
         return el
+
+    def add_child(self, name, value=None, **kwargs):
+        return self.nest(GenericNode(name, value, **kwargs))
+
+
+class MessagingResponse(TwiML):
+    """ <Response> TwiML for Messages """
+
+    def __init__(self, **kwargs):
+        super(MessagingResponse, self).__init__(**kwargs)
+        self.name = 'Response'
+
```

Comment:
What's the reason we're defining another `MessageResponse` class here? Looks like it already exists along with `VoiceResponse` and `FaxResponse` in the twiml directory